routes/swagger.yaml

```diff
@@ -78,7 +77,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/bodies/request'
+              $ref: '../schemas/curation.json#/definitions/revisions'
```

Comment:
if we use the relative paths (../) we should have a base path configured --> https://swagger.io/docs/specification/api-host-and-base-path/
 (not sure this is needed)